pkgs/tools/games/minecraft/optifine/update.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i python3 -p python3 python3.pkgs.requests python3.pkgs.lxml
+
+from lxml import html
+import json
+import os.path
+import re
+import requests
+import subprocess
+
+def nix_prefetch_sha256(name):
+    return subprocess.run(['nix-prefetch-url', '--type', 'sha256', 'https://optifine.net/download?f=' + name], capture_output=True, text=True).stdout.strip()
```

Comment:
Should probably include `nix` in the shell so that `nix-prefetch-url` will always be available.